python/tvm/octo/utils/target_info.py

```diff
@@ -0,0 +1,156 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+# pylint: disable=invalid-name, wrong-import-position
+"""Utility functions for finding information about current device."""
+import os
+import re
+import sys
+import psutil
+import subprocess
+import shutil
+import tvm
+
+
+def get_llvm_target() -> tvm.target.Target:
```

Comment:
tests for these functions? can unittest.mock subprocess.check_output to capture various outputs

Reply:
Given the amount of string processing that happens in here, we really should have a test. Mocking would be a good approach

Reply:
I totally agree but would like to defer for now. I dont even think we'll be using the auto llvm feature for a while and writing a good test suite would be time consuming. If we ever start using this feature we can make sure it is fully tested.